values/vault-operator/vault-operator-raw.gotmpl

```diff
@@ -78,10 +80,19 @@ resources:
         - name: vault-file
           persistentVolumeClaim:
             claimName: vault-file
-
+        {{- if $v._derived.untrustedCA }}
+        - name: root-ca
+          secret:
+            secretName: root-ca
+        {{- end }}
       volumeMounts:
         - name: vault-file
           mountPath: /vault/file
+      {{- if $v._derived.untrustedCA }}
+        - name: 'root-ca'
+          mountPath: '/etc/ssl/certs'
```

Comment:
So this overwrites the entire CA root chain, which might have unforeseen consequences, like no traffic possible anymore to any other service that is not using our CA. Can we not create a job somehow that appends it?

Reply:
Instead of job maybe we could use `subPath`
Before doing that can we figure it out the use case when vault needs to communicate with outside world?

Reply:
fixed with subPath